README.md

```diff
@@ -1,3 +1,106 @@
 # Tech-Writing-Project
 Class repository
 Getting started!
+
+
+----
+
```

Comment:
Here are some rough instructions on installing and running the site on your machine.
Please forgive the roughness, I had limited time, and can walk you both through it to get your machine setup. It will work on macOS and linux, and should work on Windows too.

Reply:
Nice job! I tested out the changes on my mac, and was able to successfully open up the website at "localhost:8080" in my browser.